vignettes/Variation.Rmd

```diff
@@ -1,66 +1,90 @@
 ```{r}
-# A small population
-p <- get_parameters(list(
-  human_population = 1000,
+# A small population 
+simparams <- get_parameters(list(
+  human_population = 100,
   individual_mosquitoes = FALSE
 ))
-p <- set_equilibrium(p, 2)
-small_o <- run_simulation(365, p)
-p <- get_parameters(list(
+
+simparams <- set_equilibrium(simparams, init_EIR = 20)
+
+# A larger population 
+simparams <- get_parameters(list(
   human_population = 10000,
   individual_mosquitoes = FALSE
 ))
-p <- set_equilibrium(p, 2)
-big_o <- run_simulation(365, p)
-
-plot_grid(
-  ggplot(small_o) + geom_line(aes(timestep, n_detect_730_3650)) + ggtitle('n_detect at n = 1,000'),
-  ggplot(small_o) + geom_line(aes(timestep, p_detect_730_3650)) + ggtitle('p_detect at n = 1,000'),
-  ggplot(big_o) + geom_line(aes(timestep, n_detect_730_3650)) + ggtitle('n_detect at n = 10,000'),
-  ggplot(big_o) + geom_line(aes(timestep, p_detect_730_3650)) + ggtitle('p_detect at n = 10,000')
-)
+
+simparams <- set_equilibrium(simparams, init_EIR = 20)
 ```
 
-The n_detect output shows the result of sampling individuals who would be detected by microscopy.
 
-While the p_detect output shows the sum of probabilities of detection in the population.
+## Simulations
+
+The `n_detect_730_3650` output below shows the total number of individuals in the age group rendered (here, 730-3650 timesteps or 2-10 years) who have microscopy-detectable malaria. Notice that the output is smoother at a higher population.
 
-Notice that the p_detect output is slightly smoother. That's because it forgoes the sampling step. At low population sizes, p_detect will be smoother than the n_detect counterpart.
+```{r, fig.align='center',fig.height=4, fig.width=10}
+# A small population
+output_small_pop <- run_simulation(365, simparams)
+
+# A larger population
+output_big_pop <- run_simulation(365, simparams)
+
+# Plotting 
+par(mfrow = c(1,2))
+plot_n_detect(output_small_pop); title('n_detect at n = 100')
+plot_n_detect(output_big_pop, ylab = FALSE); title('n_detect at n = 10,000')
+```
 
 ## Estimating variation 
 
-We can estimate the variation in the number of detectable cases by repeating the simulation several times...
+We can estimate the variation in the number of detectable cases by repeating the simulation several times with the function `run_simulation_with_repetitions()`, then plotting the IQR and 95% confidence intervals. Running multiple simulations is particularly important for simulations of a small population size, due to the disproportionate impact of stochasticity in small populations. 
 
-```{r}
+```{r, fig.align='center',fig.height=4, fig.width=10}
 outputs <- run_simulation_with_repetitions(
-  timesteps = 365,
-  repetitions = 10,
-  overrides = p,
-  parallel=TRUE
+  timesteps = 365, # We will run each simulation for 365 timesteps.
+  repetitions = 10, # The simulation will be run 10 times
```

Comment:
Use "simulation" for either the overall run or each individual run. 
`# Each repetition will be run for 365 days`
`# The simulation will be run 10 times`